SpatialGDK/Extras/schema/spawndata.schema

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
+
+import "improbable/vector3.schema";
+
+type Rotator {
+    float pitch = 1;
+    float yaw = 2;
+    float roll = 3;
+}
+
+component SpawnData {
+    id = 100001;
+    Rotator rotation = 1;
```

Comment:
Could you add comments into this schema file regarding why this component is the way it is.